driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/execve.bpf.c

```diff
@@ -239,9 +239,10 @@ int BPF_PROG(t1_execve_x,
 	auxmap__store_s32_param(auxmap, (s32)loginuid);
 
 	/* Parameter 20: flags (type: PT_FLAGS32) */
-	/// TODO: still to implement! See what we do in the old probe.
+	/// TODO: we still have to manage `exe_writable` flag.
 	u32 flags = 0;
-	if(extract__exe_upper_layer(task))
+	struct inode *exe_inode = extract__exe_inode_from_task(task);
```

Comment:
we miss the same fix in the ARM/s390-dedicated tracepoint `sched_process_exec`

Reply:
🤦‍♀️ right, ty!